gateway/src/test/java/io/zeebe/ResponseMapperTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.zeebe.broker.client.api.commands.BrokerInfo;
+import io.zeebe.broker.client.api.commands.PartitionInfo;
+import io.zeebe.broker.client.api.commands.Topology;
+import io.zeebe.factories.TopologyFactory;
+import io.zeebe.gateway.protocol.GatewayOuterClass;
+import io.zeebe.gateway.protocol.GatewayOuterClass.HealthResponse;
+import io.zeebe.gateway.protocol.GatewayOuterClass.Partition;
+import java.util.List;
+import org.junit.Test;
+
+public class ResponseMapperTest {
+
+  @Test
+  public void shouldTestHealthCheckMapping() {
+    final ResponseMapper responseMapper = new ResponseMapper();
+    final Topology topology = new TopologyFactory().getFixture();
+    final List<BrokerInfo> brokers = topology.getBrokers();
+
+    final HealthResponse response = responseMapper.toResponse(topology);
+
+    assertThat(response.getBrokersCount()).isEqualTo(brokers.size());
```

Comment:
We should come up with a better testing approach to allow us to write small readable tests which can be used to test different scenarios and input data, this should be tackled in #1124.